java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_2/StringValidator.java

```diff
@@ -0,0 +1,36 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class StringValidator extends AccountValidator {
```

Comment:
Why does StringValidator extend AccountValidator? Does it really need to override its method? I think it might be better to use some interface here and make it common for all validators.

Reply:
Made AccountValidator interface.